src/core/inlines.js

```diff
@@ -131,13 +131,20 @@ function inlineRefMatches(matched) {
 
 /**
  * @param {string} matched
+ * @param {Text} text
  */
-function inlineXrefMatches(matched) {
+function inlineXrefMatches(matched, text) {
   // slices "{{" at the beginning and "}}" at the end
-  const ref = matched.slice(2, -2).trim();
-  return ref.startsWith("\\")
-    ? matched.replace("\\", "")
-    : idlStringToHtml(norm(ref));
+  const ref = norm(matched.slice(2, -2).trim());
```

Comment:
`norm` does the trim:
```suggestion
  const ref = norm(matched.slice(2, -2));
```